umbraco/umbraco.d.ts

```diff
@@ -0,0 +1,116 @@
+// Type definitions for Umbraco v7.5.0
```

Comment:
This should be added to `index.d.ts` instead. Having `foo/foo.d.ts` was the old style, but now we use `index.d.ts` instead.